charts/config-validator/templates/pod/_config-validator.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Versisn 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- define "spec.container.configValidator" }}
+- name: config-validator
+  image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+  # -policyPath        https://github.com/forseti-security/policy-library/tree/master/policies
+  # -policyLibraryPath https://github.com/forseti-security/policy-library/tree/master/lib
```

Comment:
Can we delete these comments?

Reply:
We can.  I had left them there for my reference.